cmd/kubeadm/app/phases/upgrade/health.go

```diff
@@ -158,7 +164,8 @@ func createJob(client clientset.Interface, cfg *kubeadmapi.ClusterConfiguration)
 		return true, nil
 	})
 	if err != nil {
-		return errors.Wrapf(lastError, "could not create a Job with the prefix %q in the namespace %q", prefix, ns)
+		// Return the error as a warning for this preflight check
+		return errors.Wrapf(lastError, "could not create a Job with the prefix %q in the namespace %q", prefix, ns), nil
```

Comment:
IMO, it shouldn't be a warning. the apiserver should accept this request.

Reply:
updated.